projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/deploy/JobImageDeployerV2.java

```diff
@@ -229,10 +229,76 @@ private ActualDataJobDeployment updateCronJob(
           DeploymentModelConverter.toActualJobDeployment(
               desiredDataJobDeployment, desiredDeploymentVersionSha, lastDeployedDate);
     }
-
+    checkDefaultDeploymentResources(actualJobDeployment);
```

Comment:
I think we should incorporate this logic into the `DeploymentModelConverter.toActualJobDeployment()` and get the resources from `KubernetesService.CronJob desiredCronJob` since they are already set.